source/DataFile.cpp

```diff
@@ -23,15 +23,17 @@ using namespace std;
 
 
 // Constructor, taking a file path (in UTF-8).
-DataFile::DataFile(const string &path)
+DataFile::DataFile(const string &path, shared_ptr<ConditionsStore> store0):
+	store(store0)
```

Comment:
```suggestion
: store(store0)
```

I might be super stupid, but isnt this how you normally do it?

Reply:
You're right. I just skimmed through some other Endless Sky code, and it seems to be that way.

Reply:
Actually, there's a tab before the `:` in the other code.

Reply:
whoops forgot that 💀

Reply:
I've corrected all instances of this particular style problem, and there were several.